src/formatDate.js

```diff
@@ -8,7 +8,46 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arrayFromDate = date.split(fromFormat[fromFormat.length - 1]);
+  const arrForResulf = [];
```

Comment:
There is a typo in the variable name `arrForResulf`. It should be `arrForResult` to reflect its purpose correctly and avoid confusion.